src/utils/helpers.js

```diff
@@ -0,0 +1,12 @@
+export function setButtontext(
+  btn,
+  isLoading,
+  defaultText = "Save",
+  loadingText = "Saving..."
+) {
+  if (isLoading) {
+    btn.innerHTML = "Loading...";
+  } else {
+    btn.innerHTML = "Submit";
```

Comment:
You have `defaultText` and `loadingText`. Please, use them.

```js

  if (isLoading) {
    btn.innerHTML = loadingText;
  } else {
    btn.innerHTML = defaultText;
  }
```


And you can reassign them when you call `setButtontext` for the confirmation modal if you need another text: `Deleting...` and so on